src/test/java/subway/LineAcceptanceTest.java

```diff
@@ -0,0 +1,272 @@
+package subway;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.restassured.RestAssured;
+import io.restassured.path.json.JsonPath;
+import io.restassured.response.ExtractableResponse;
+import io.restassured.response.Response;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.context.SpringBootTest.WebEnvironment;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.test.context.jdbc.Sql;
+import org.springframework.test.context.jdbc.Sql.ExecutionPhase;
+
+@DisplayName("지하철 노선 관련 기능")
+@Sql(scripts = {"/truncate.sql"}, executionPhase = ExecutionPhase.BEFORE_TEST_METHOD)
+@SpringBootTest(webEnvironment = WebEnvironment.DEFINED_PORT)
+public class LineAcceptanceTest {
+    /**
+     * Given: 새로운 지하철 노선 정보를 입력하고,
+     * When: 관리자가 노선을 생성하면,
+     * Then: 해당 노선이 생성되고 노선 목록에 포함된다.
+     */
+    @DisplayName("지하철 노선을 생성한다")
+    @Test
+    void createLine() {
+        // Given
+        Long firstStationId = requestCreateStation("지하철역")
+                                        .jsonPath()
+                                        .getObject("id", Long.class);
+        Long secondStationId = requestCreateStation("새로운지하철")
+                                        .jsonPath()
+                                        .getObject("id", Long.class);
+
+        // When
+        Map<String, Object> params = new HashMap<>();
+        params.put("name", "신분당선");
+        params.put("color", "bg-red-600");
+        params.put("upStationId", firstStationId);
+        params.put("downStationId", secondStationId);
+        params.put("distance", 10);
+        ExtractableResponse<Response> response = requestCreateLine(params);
+
+        // Then
+        assertThat(response.statusCode()).isEqualTo(HttpStatus.CREATED.value());
+        List<Long> stationIds = response.jsonPath().getList("stations.id", Long.class);
+        System.out.println("stationIds = " + stationIds);
+        assertThat(stationIds).containsExactlyInAnyOrder(firstStationId, secondStationId);
```

Comment:
```suggestion
        /** 노선 이름 검증**/
        assertThat(stationIds).containsExactlyInAnyOrder(firstStationId, secondStationId);
```

노선 이름에 대한 검증과, 지하철역에 대한 검증만 있어도 충분할 것 같아요!